src/hide-discord-sidebar.css

```diff
@@ -1,4 +1,8 @@
 /* .hide-dis-bar class on body indicates that extension is active */
+body.hide-dis-bar div[class*="toolbar-"] {

```

Comment:
Hey @bokunodev, thanks for the PR. However, the change you introduced actually is the same as what is existing in the highlighted lines below. Maybe the extension's CSS wasn't loaded previously when you faced the issue? You can try CTRL/CMD+SHIFT+R to do a hard refresh and I think the problem itself might be fixed by the styles loading. Let me know if it happens otherwise.

![image](https://user-images.githubusercontent.com/20418855/210194963-ab0e1bab-51c0-47bb-9cfd-1ed0fa278e7c.png)